app/scripts/directives/configEditor.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+angular.module('openshiftConsole')
+  .directive('configEditor', function(APIService, DataService) {
```

Comment:
If you can use the `copyToClipBoard` directive I would move this to a component.

Reply:
I cannot use the copyToClipBoard directive.

Reply:
Why can't it be used?

Reply:
It cannot be used without displaying a textarea, and the textarea cannot be editable.
If this field does not need to be editable, then I could re-use it, but I recall a conversation where it was stressed that this should be editable?